astropy/coordinates/sky_coordinate.py

```diff
@@ -716,11 +716,12 @@ def transform_to(self, frame, merge_attributes=True):
         return self.__class__(new_coord, **frame_kwargs)
 
     def apply_space_motion(self, new_obstime=None, dt=None):
-        """
+        """Compute the position to a new time using the velocities.
+
         Compute the position of the source represented by this coordinate object
         to a new time using the velocities stored in this object and assuming
         linear space motion (including relativistic corrections). This is
-        sometimes referred to as an "epoch transformation.".
+        sometimes referred to as an "epoch transformation."
```

Comment:
I'd swap the closing " with the period, i.e., "... transformation". (I'm very fond of putting the closing quotes on the outside, but it makes sense most if properly typeset, since then the period is just underneath the quotes.)